xml/System.Diagnostics/Process.xml

```diff
@@ -1321,7 +1321,7 @@ The following code example creates a process that prints a file. It sets the <xr
   
  On any particular computer, the identifier of a process is unique. <xref:System.Diagnostics.Process.GetProcessById%28System.Int32%29> returns one process at most. If you want to get all the processes running a particular application, use <xref:System.Diagnostics.Process.GetProcessesByName%28System.String%29>. If multiple processes exist on the computer running the specified application, <xref:System.Diagnostics.Process.GetProcessesByName%28System.String%29> returns an array containing all the associated processes. You can query each of these processes in turn for its identifier. The process identifier can be viewed in the `Processes` panel of the Windows Task Manager. The `PID` column displays the process identifier that is assigned to a process.  
   
-   
+ Please note that the `processId` parameter is a signed `int` although the underlying Windows API will use an unsigned `DWORD` for similar APIs. This is for historical reasons.
```

Comment:
- "Please note that" can be deleted. So the sentence should begin with "The \`processId\` parameter"
- comma before although
- the signed `int` should be language-independent: is a signed \`int\` --> is an \<xref:System.Int32> (a 32-bit signed integer)
- will use --> uses
- DWORD by definition is unsigned, so: an unsigned \`DWORD\` --> a \`DWORD\` (an unsigned 32-bit integer)

The same note should also be added on line 1396, since both overloads have an Int32 parameter.